pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -97,6 +97,10 @@
         %div
           %a{onclick: "return showVideo_mainvideo();", style: "cursor: pointer"}
             %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px"}
+      - elsif entry[:type] == "celebrate_bhm"
+        %a.herohocbutton.bhmbutton{href: "/blackhistorymonth"}
+          %i.fa.fa-arrow-circle-o-right
+          %span Celebrate black history month with us!
```

Comment:
Should "Black History Month" be capitalized?

Reply:
That was an oversight, good catch!